external-crates/move/crates/move-cli/tests/sandbox_tests/clever_errors/tests/AModuleTests.move

```diff
@@ -0,0 +1,55 @@
+#[test_only]
+module std::AModuleTests {
```

Comment:
For my sanity, can you split this into two files, one for failing tests and one for passing tests?

Reply:
Will try!

Reply:
Split!